js_modules/dagster-ui/packages/ui-core/src/app/AppProvider.tsx

```diff
@@ -181,14 +181,14 @@ export const AppProvider = (props: AppProviderProps) => {
     <AppContext.Provider value={appContextValue}>
       <WebSocketProvider websocketClient={websocketClient}>
         <ApolloProvider client={apolloClient}>
-          <AssetLiveDataProvider>
-            <PermissionsProvider>
-              <BrowserRouter basename={basePath || ''}>
-                <GlobalStyleProvider />
-                <CompatRouter>
-                  <TimeProvider>
-                    <CodeLinkProtocolProvider>
-                      <WorkspaceProvider>
+          <PermissionsProvider>
+            <BrowserRouter basename={basePath || ''}>
+              <GlobalStyleProvider />
+              <CompatRouter>
+                <TimeProvider>
+                  <CodeLinkProtocolProvider>
+                    <WorkspaceProvider>
+                      <AssetLiveDataProvider>
```

Comment:
AssetLiveDataProvider renders the SelectionHealthDataProvider which depends on WorkspaceProvider so I had to move AssetLiveProvider inside of it.